packages/expo-asset/src/resolveAssetSource.web.ts

```diff
@@ -38,29 +37,6 @@ function getDevServerURL(): string | null {
   return _serverURL;
 }
 
-function _coerceLocalScriptURL(scriptURL: string | undefined | null): string | null {
-  if (scriptURL) {
-    if (scriptURL.startsWith('assets://')) {
-      // android: running from within assets, no offline path to use
-      return null;
-    }
-    scriptURL = scriptURL.substring(0, scriptURL.lastIndexOf('/') + 1);
-    if (!scriptURL.includes('://')) {
-      // Add file protocol in case we have an absolute file path and not a URL.
-      // This shouldn't really be necessary. scriptURL should be a URL.
-      scriptURL = 'file://' + scriptURL;
-    }
-  }
-  return null;
-}
-
-function getScriptURL(): string | null {
-  if (_scriptURL === undefined) {
-    _scriptURL = _coerceLocalScriptURL(getSourceCodeScriptURL());
-  }
-  return _scriptURL;
-}
-
```

Comment:
`_coerceLocalScriptURL` always returns `null` causing `getScriptURL` to also always return null.
@EvanBacon Do you know whether this was ever intentional?